activiti-services/activiti-services-events/src/main/java/org/activiti/services/events/MessageProducerCommandContextCloseListener.java

```diff
@@ -0,0 +1,48 @@
+package org.activiti.services.events;
+
+import java.util.List;
+
+import org.activiti.engine.impl.context.Context;
+import org.activiti.engine.impl.interceptor.CommandContext;
+import org.activiti.engine.impl.interceptor.CommandContextCloseListener;
+import org.activiti.services.api.events.ProcessEngineEvent;
+import org.springframework.messaging.support.MessageBuilder;
+
+public class MessageProducerCommandContextCloseListener implements CommandContextCloseListener {
+
+    public static final String PROCESS_ENGINE_EVENTS = "processEngineEvents";
+
+    private final ProcessEngineChannels producer;
+
+    public MessageProducerCommandContextCloseListener(ProcessEngineChannels producer) {
+        this.producer = producer;
+    }
+
+    @Override
+    public void closed(CommandContext commandContext) {
+        CommandContext currentCommandContext = Context.getCommandContext();
+        @SuppressWarnings("unchecked")
+        List<ProcessEngineEvent> events = (List<ProcessEngineEvent>) currentCommandContext
+                                                                                          .getAttribute(PROCESS_ENGINE_EVENTS);
```

Comment:
Thank you for your advice. I fixed.
https://github.com/Activiti/Activiti/pull/1424/commits/5d74bf002f8b8537147b9b0096feb4bc415b659f